pkgs/by-name/de/deno/patches/unvendor-sqlite.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Cargo.toml b/Cargo.toml
```

Comment:
Is this patch unused or did I just miss something?

Reply:
It's used, but you made me realize this is not strictly related to tests, I just think it would be a good idea. Just like how we unvendored `libffi` after it caused a build failure a few months ago: https://github.com/NixOS/nixpkgs/pull/368102

Should I drop this change?